src/administration/snapshot-and-restore.md

```diff
@@ -80,11 +81,11 @@ Snapshots may be triggered by calling the CLI from an automated system such as J
 
 > **note**
 >
-> Snapshots using cron requires you to [install the CLI in your container and set up an API token](/gettingstarted/cli/api-tokens.md).
+> Automated snapshots using cron requires you to [get an API token and install the CLI in your application container](/gettingstarted/cli/api-tokens.md).
```

Comment:
This file has moved to `/development/cli/api-tokens.md`. 

The redirect has been set-up, but since you'll have to resolve the new location conflict, you may want to update this link too.